mk/spksrc.wheel-env.mk

```diff
@@ -90,7 +86,9 @@ install_python_wheel:
 		else \
 			for w in *.whl; do \
 				$(MSG) Copying to wheelhouse: $$(echo $$w | sed -E "s/(.*linux_).*(\.whl)/\1$(PYTHON_ARCH)\2/") ; \
-				cp -f $$w $(STAGING_INSTALL_WHEELHOUSE)/$$(echo $$w | sed -E "s/(.*linux_).*(\.whl)/\1$(PYTHON_ARCH)\2/") ; \
+				cp -f $$w $(STAGING_INSTALL_WHEELHOUSE)/$$(echo $$w \
+					| sed -E "s/(.*-).*(linux_.*\.whl)/\1\2/" \
+					| sed -E "s/(.*linux_).*(\.whl)/\1$(PYTHON_ARCH)\2/") ; \
 			done ; \
```

Comment:
this can be simplified to (and fixes MSG output)
```
			for w in *.whl; do \
				_new_name=$$(echo $${w} | sed -E "s/(.*-).*/\1/")linux_$(PYTHON_ARCH).whl ; \
				$(MSG) Copying to wheelhouse: $${_new_name} ; \
				cp -f $${w} $(STAGING_INSTALL_WHEELHOUSE)/$${_new_name} ; \
			done ; 
```

Reply:
done, slightly adapted.